include/bluetooth/mcc.h

```diff
@@ -26,259 +34,321 @@ extern "C" {
  * @param icon_url      The URL of the Icon
  */
-typedef void (*bt_mcc_icon_url_read_cb)(struct bt_conn *conn, int err,
-					const char *icon_url);
+typedef void (*bt_mcc_icon_url_read_cb)(struct bt_conn *conn, int err, const char *icon_url);
 
-/** @brief Callback function for track changed notifications
+/**
+ * brief Callback function for track changed notifications
+ *
+ * Called when a track change is notified.
  *
- * The track changed characteristic is a special case.  It can not be
- * read or written, it can only be notified.
+ * The track changed characteristic is a special case.  It can not be read or
+ * written, it can only be notified.
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  */
 typedef void (*bt_mcc_track_changed_ntf_cb)(struct bt_conn *conn, int err);
 
 
-/** @brief Callback function for bt_mcc_read_track_title()
+/**
+ * brief Callback function for bt_mcc_read_track_title()
+ *
+ * Called when the track title is read or notified
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param title         The title of the track
  */
-typedef void (*bt_mcc_track_title_read_cb)(struct bt_conn *conn, int err,
-					   const char *title);
+typedef void (*bt_mcc_track_title_read_cb)(struct bt_conn *conn, int err, const char *title);
 
-/** @brief Callback function for bt_mcc_read_track_dur()
+/**
+ * brief Callback function for bt_mcc_read_track_duration()
+ *
+ * Called when the track duration is read or notified
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param dur           The duration of the track
  */
-typedef void (*bt_mcc_track_dur_read_cb)(struct bt_conn *conn, int err,
-					 int32_t dur);
+typedef void (*bt_mcc_track_duration_read_cb)(struct bt_conn *conn, int err, int32_t dur);
 
-/** @brief Callback function for bt_mcc_read_track_position()
+/**
+ * brief Callback function for bt_mcc_read_track_position()
+ *
+ * Called when the track position is read or notified
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param pos           The Track Position
  */
-typedef void (*bt_mcc_track_position_read_cb)(struct bt_conn *conn, int err,
-					      int32_t pos);
+typedef void (*bt_mcc_track_position_read_cb)(struct bt_conn *conn, int err, int32_t pos);
 
-/** @brief Callback function for bt_mcc_set_track_position()
+/**
+ * brief Callback function for bt_mcc_set_track_position()
+ *
+ * Called when the track position is written
```

Comment:
Given the of the function, shouldn't this be
```suggestion
 * Called when the track position is set
```

Reply:
This of course applies to all the `set` functions.

Reply:
That is more consistent with the function name, yes.
I used "written" since it is a GATT write, but if so it wold be better to change the function name as well - wait with that until the jury is back on function naming.

Fixed